tests/python_package_test/test_dask.py

```diff
@@ -1084,6 +1084,45 @@ def test_dask_methods_and_sklearn_equivalents_have_similar_signatures(methods):
         assert dask_params[param].default == sklearn_params[param].default, error_msg
 
 
+@pytest.mark.parametrize('task', tasks)
+def test_training_succeeds_when_data_is_dataframe_and_label_is_column_array(
+    task,
+    client,
+    listen_port
```

Comment:
@jameslamb Should `listen_port` be removed here given that #3994 is merged now?

Reply:
yes it should